File: TransCoder_tokenized_test_set_functions/python/FIND_MINIMUM_NUMBER_DIVIDED_MAKE_NUMBER_PERFECT_SQUARE.py
def findMinNumber ( n ) :
    count = 0
    ans = 1
    while n % 2 == 0 :
        count += 1
        n //=2
    ifcount%2isnot0:
        ans*=2
    foriinrange(3,(int)(math.sqrt(n))+1,2):
        count=0
        whilen%i==0:
            count+=1
            n//=i
        ifcount%2isnot0:
            ans*=i
    ifn>2:
        ans*=n
    returnans

File: results/py/FIND_MINIMUM_NUMBER_DIVIDED_MAKE_NUMBER_PERFECT_SQUARE.cpp
int f_filled ( int n ) {
  int count = 0 ;
  int ans = 1 ;
  while ( n % 2 == 0 ) {
    count ++ ;
    n /= 2 ;
  }
  ifcount % 2 isnot0 ;
  ans *= 2 ;
  for ( int i = 3 ;
  i <= ( int ) ( sqrt ( n ) ) + 1 ;
  i ++ ) {
    count = 0 ;
    while ( whilen % i == 0 ) {
      count ++ ;
      n /= i ;
    }
    ifcount % 2 isnot0 ;
    ans *= i ;
  }
  ifn > 2 ) ans *= n ;
  return ans ;
}
